lib/rspec/core/example_group.rb

```diff
@@ -26,7 +26,7 @@ module Core
     # your examples defined in {MemoizedHelpers} and {Pending}.
     class ExampleGroup
       extend Hooks
-
+      extend FailFastHandler
```

Comment:
Since `FailFastHandler` is completely private and doesn't have any APIs we want to expose to users, there's no need for the methods to be added to `ExampleGroup` like this.  After all, it's not a property of the example group class whether or not the failures number has been met.  I'd prefer it if you just made the methods exposed by `FailFastHandler` module methods and called them as e.g. `FailFastHandler.failures_number_met?` w/o extending the module.